libraries/state_history/trace_converter.cpp

```diff
@@ -0,0 +1,55 @@
+#include <eosio/state_history/serialization.hpp>
+#include <eosio/state_history/trace_converter.hpp>
+
+namespace eosio {
+namespace state_history {
+
+using eosio::chain::packed_transaction;
+using eosio::chain::plugin_exception;
+
+bool is_onblock(const transaction_trace_ptr& p) {
+   if (p->action_traces.size() != 1)
```

Comment:
Should be:
```cpp
if (p->action_traces.empty())
```

The `is_onblock` in `trace_api/chain_extraction.hpp` also needs a similar correction.

Reply:
changed